terraform/statuscake/dev.bk.vars

```diff
@@ -1,4 +1,2 @@
 storage_account_name = "s146d01sgtfstate"
-key="tta_statuscake.dev.terraform.tfstate"
```

Comment:
Can we remove the `terraform/` and `tests/` stuff from the diff, it's not changing and it's unrelated to your PR

Reply:
They had linting "errors" which wouldn't pass the commit until resolved.

Reply:
Urgh, ok must have been merged without CI passing originally - still worth cherry-picking these changes into another PR IMO